modules/status.js

```diff
@@ -0,0 +1,55 @@
+class Status {
+  constructor(completed, index) {
+    this.completed = completed;
+    this.index = index;
+  }
+
+  Set = () => {
+    const tasks = JSON.parse(localStorage.getItem('to_do_list'));
+    tasks.forEach((task) => {
+      if (task.index === this.index) {
+        task.completed = this.completed;
+      }
+    });
```

Comment:
I believe you should change the code with the suggestion. because it will reduce the time complexity.
```suggestion
    tasks[this.index - 1].completed = this.completed;
```